src/utils/getAccessTabData.ts

```diff
@@ -5,6 +5,21 @@ import { IAccessItem } from '../interfaces/searchResponse.interface';
 import { getOrganisationDetails } from './getOrganisationDetails';
 import { addSpaces, capitalizeWords } from './formatAggregationResponse';
 
+const rolePrecedence = ['owner', 'pointOfContact', 'custodian', 'distributor', 'originator'];
+
+const combineAndSortContacts = (contactForResource: any[], contact: any[]): any[] => {
+  const contactForResourceArray = contactForResource ?? [];
+  const contactArray = contact ?? [];
+  const combined = [...contactForResourceArray, ...contactArray];
+  combined.sort((a, b) => {
+    const roleIndexA = rolePrecedence.indexOf(a.role.toLowerCase());
+    const roleIndexB = rolePrecedence.indexOf(b.role.toLowerCase());
```

Comment:
rolePrecedence has camelcase but looking for index of role  after transforming them to lowercase. how does it works?

Reply:
I have resolved by removing camelcase of pointofcontact.